src/test/java/CarRunnerTest.java

```diff
@@ -0,0 +1,43 @@
+import static org.assertj.core.api.Assertions.assertThat;
+
+import domain.Car;
+import domain.FixedNumberGenerator;
+import domain.RandomNumberGenerator;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.DisplayNameGeneration;
+import org.junit.jupiter.api.DisplayNameGenerator;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+@DisplayNameGeneration(DisplayNameGenerator.ReplaceUnderscores.class)
+public class CarRunnerTest {
+
+  @ParameterizedTest
+  @ValueSource(ints = {1, 2, 3})
+  @DisplayName("스피드가 3 이하면 자동차는 움직이지 않는다")
+  void notMoveWhenThreeOrLess(int speed) {
+    Car car = new Car("Slow", new FixedNumberGenerator(speed));
+    car.move();
+    assertThat(car.getPosition()).isEqualTo(0);
+  }
```

Comment:
```suggestion
  @ParameterizedTest
  @ValueSource(ints = {1, 2, 3})
  @DisplayName("스피드가 3 이하면 자동차는 움직이지 않는다")
  void notMoveWhenThreeOrLess(int speed) {
    // given
    Car car = new Car("Slow", new FixedNumberGenerator(speed));
    
    // when
    car.move();
    
    // then
    assertThat(car.getPosition()).isEqualTo(0);
  }
```

정상 통과 테스트의 경우에는 `given - when - then`에 대해 학습하면 좋아보여요~

Reply:
👍가독성이 더 좋네요

Reply:
요 부분은 지금 당장은 가독성이 살짝 좋아지는 연습용이지만, 나중에 팀플젝이나 더 복잡한 미션을 하면서 테스트 코드를 작성하게 된다면 가독성이 매우 좋아지는 역할을 하게 됩니답